### PR TITLE
LibJS: One normative update + big nanosecond addition fix

### PR DESCRIPTION
**Pull request by davidot**
I think this is my first normative update so let me know if the message needs more explanation

(Github doesn't like this long branch name...)

Fixes one of the comments mentioned in #15525

### CODE REVIEW

File: Userland/Libraries/LibJS/Tests/builtins/Temporal/Now/Now.plainDate.js [hunk 1]
```diff
@@ -34,6 +34,21 @@ describe("correct behavior", () => {
...
+        [86400000000000, -86400000000000, 86400000000001, 86400000000002].forEach(offset => {
+            const calendar = new Temporal.Calendar("iso8601");
+            const timeZone = {
+                getOffsetNanosecondsFor() {
+                    return offset;
+                },
+            };
+            expect(() => Temporal.Now.plainDate(calendar, timeZone)).toThrowWithMessage(
+                RangeError,
+                "Invalid offset nanoseconds value, must be in range -86400 * 10^9 to 86400 * 10^9"
```
Comment (linusg) [inline Userland/Libraries/LibJS/Tests/builtins/Temporal/Now/Now.plainDate.js:48]: This is a lie now

**Review by linusg — CHANGES_REQUESTED** [review]

File: Userland/Libraries/LibJS/Tests/builtins/Temporal/Now/Now.plainDate.js (hunk 1 above)
Comment (davidot) [reply Userland/Libraries/LibJS/Tests/builtins/Temporal/Now/Now.plainDate.js]: Yeah I thought we could just say the range is exclusive now but it isn't very clear.
I added +1 and -1, but I'm not super happy with it.
If you have a better suggestion let me know!

Comment (davidot) [reply Userland/Libraries/LibJS/Tests/builtins/Temporal/Now/Now.plainDate.js]: Hold on splitting the error

Comment (linusg) [reply Userland/Libraries/LibJS/Tests/builtins/Temporal/Now/Now.plainDate.js]: updated message seems fine :)

**Pull request merged**